live-site/components/judging-section/judging-selection/JudgingSelection.styles.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components';
+import { max } from '../../../../shared-ui/lib/responsive';
+import { colors } from '../../../../shared-ui/style/colors';
+import { H2 } from '../../../../shared-ui/style/typography';
+
+const StyledJudgingSelection = styled.div`
+  padding: 10em;
```

Comment:
change to padding: 10em 0; 

Its already centered so we don't need all the extra padding on the left and right since it causes it to be more smushed on mobile

Just be sure to stack the buttons on mobile. Also, to please address the button style inconsistencies that @mike10911 pointed out
<img width="489" alt="Screenshot 2023-01-05 at 6 36 50 PM" src="https://user-images.githubusercontent.com/59738880/210900352-cd62702f-a52c-4ac8-8240-829e8a7eddaf.png">